packages/frontend/src/components/chart/configure/state/getEndpoint.ts

```diff
@@ -1,7 +1,30 @@
-export function getEndpoint(element: HTMLElement) {
-  const endpoint = element.dataset.endpoint
+export function getInitialEndpoint(
+  type: 'tvl' | 'activity',
+  chart: HTMLElement,
+  selectedToken?: HTMLInputElement,
+) {
+  if (type === 'activity') {
+    return getActivityEndpoint(chart)
+  }
+
+  if (selectedToken) {
+    return getTvlEndpoint(selectedToken)
+  }
+  return getTvlEndpoint(chart)
+}
+
+export function getTvlEndpoint(element: HTMLElement) {
+  const endpoint = element.dataset.tvlendpoint
```

Comment:
if you use `data-tvl-endpoint` then you can access it as `element.dataset.tvlEndpoint`. Same for activity

Reply:
Oh, that's definitely useful